packages/datatrak-web/src/views/AccountSettingsPage/RequestCountryAccessSection/RequestCountryAccessForm.tsx

```diff
@@ -0,0 +1,244 @@
+  gap: 1.25rem;
+  grid-auto-flow: column;
+  grid-template: auto auto / auto;
+
+  ${theme.breakpoints.up('sm')} {
+    grid-template: auto / 1fr 1fr;
+  }
+
+  .MuiFormLabel-root {
+    color: ${props => props.theme.palette.text.primary};
+    font-weight: ${theme.typography.fontWeightMedium};
+  }
+
+  // Fix labels appearing over hamburger menu drawer
+  .MuiInputLabel-outlined {
+    z-index: auto;
+  }
+`;
+
+const CountryChecklistWrapper = styled(MuiBox)`
+  display: block flex;
+  flex-direction: column;
+  height: 18.625rem;
+`;
+
+/** Matches styling of .MuiFormLabel-root in ui-components TextField */
+const StyledFormLabel = styled(MuiFormLabel)`
+  font-size: 0.9375rem;
+  line-height: 1.125rem;
+  margin-bottom: 3px;
+`;
+
+const CountryChecklist = styled.fieldset`
+  margin: 0;
+  padding-block: 0;
+
+  border-radius: 0.1875rem;
+  border: 1px solid ${props => props.theme.palette.grey[400]};
+  block-size: 100%;
+  overflow-y: scroll; /* fallback */
+  overflow-block: scroll;
+  padding-inline: 0.87rem;
+
+  :disabled {
+    background-color: #f5f5f5;
+  }
+`;
+
+const StyledCheckbox = styled(Checkbox)`
+  margin-block: 0;
+
+  .MuiFormControlLabel-root {
+    inline-size: 100%;
+  }
+
+  .MuiFormControlLabel-label {
+    font-size: 0.875rem;
+    inline-size: 100%;
+    line-height: 1.125rem;
+  }
+`;
+
+const StyledBox = styled(MuiBox)`
+  display: block flex;
+  flex-direction: column;
+  gap: 1.25rem;
+`;
+
+// Usage of this component has inline styling. See there for explanation.
+const StyledFormInput = styled(FormInput)`
+  flex-grow: 1;
+  margin: 0;
+
+  .MuiInputBase-root {
+    align-items: start;
+    block-size: 100%;
+    max-block-size: 100%;
+  }
+
+  .MuiInputBase-input {
+    box-sizing: border-box;
+  }
+`;
+
+interface RequestCountryAccessFormFields {
+  countryIds: Country['id'][];
+  reasonForAccess?: string;
+}
+
+export const RequestCountryAccessForm = () => {
+  const { project } = useCurrentUser();
+  const projectCode = project?.code;
+
+  const queryResult = useCountryAccessList();
+  const countries: TupaiaWebCountryAccessListRequest.ResBody = queryResult.data ?? [];
+  const accessListIsLoading = queryResult.isLoading;
+
+  const { mutate: requestCountryAccess, isLoading: requestIsLoading } = useRequestProjectAccess({
+    onError: error =>
+      errorToast(error?.message ?? 'Sorry, couldn’t submit your request. Please try again'),
+    onSettled: () => reset(),
+    onSuccess: response => {
+      successToast(response.message);
+      rerenderCountryChecklist();
+    },
+  });
+
+  /**
+   * When React Hook Form resets the form state, boxes get correctly unchecked, but the MUI
+   * component fails to reflect this until its next render (usually when the user interacts with it,
+   * which makes the checkbox feel broken). setCountryCheckList serves merely as an "automatic
+   * kicking machine" to force a re-render.
+   */
+  const [countryChecklistKey, setCountryChecklistKey] = useState(Date.now());
```

Comment:
I expect this might be because possibly our checkbox input is an MUI one which I _think_ means it is controlled, as opposed to uncontrolled. React-hook-form typically operates uncontrolled inputs. The nicest way around this is to use a [Controller](https://legacy.react-hook-form.com/api/usecontroller/controller) component from `react-hook-form` instead